content/tracing/product_specs/metrics_namespace.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Traces metrics namespace
+kind: documentation
+---
+
+The trace metrics namespace is `trace.<name>.<metrics>{<tags>}` where
+
+* `<name>` is coming from the name of the integration ("redis", "pylons", etc.)
+* `<metrics>` is about the hits, errors or latency ("request.hits", etc.)
```

Comment:
We be more precise and exhaustive. (you can ping some of us for the real spec here).

Reply:
👌